packages/@n8n/task-runner/src/task-runner.ts

```diff
@@ -276,6 +293,7 @@ export abstract class TaskRunner {
 			taskId,
 			error,
 		});
+		console.error(`Runner failed to complete task ${taskId}`);
```

Comment:
We could log also the error message to have some idea why it failed. Tho I'm wondering if this makes sense to log all the failures, as it can fail for example if user enters invalid JS code into the Code Node.

Reply:
Decided to remove these two logs. Let's add them in a dedicated PR if/when we need them.